_posts/2016-05-26-Gerber.md

```diff
@@ -0,0 +1,306 @@
+---
+layout: post
+title: "Gerber Statistic Implementation in Rcpp and OpenMP"
+author: Yi King, Rafael Nicolas Fermin Cota, and Chris Chong - Next Level Analytics
+license: GPL (>= 2)
+tags: rcpp openmp portfolio-allocation gerber

```

Comment:
We tend to try to do the right thing with tags which aggregate across exiting tags so

- "rcpp" is redundant, please remove
- "openmp" is perfect, let's see if we can maybe think of another one
- "portfolio-allocation" and "gerber" and likely to be orphans, so please remove too